flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendSavepointTest.java

```diff
@@ -315,7 +315,7 @@ private static void restoreStdOutAndStdErr() {
 	private static ClusterClient<String> createClusterClient(String expectedResponse) throws Exception {
 		final ClusterClient<String> clusterClient = mock(ClusterClient.class);
 
-		when(clusterClient.triggerSavepoint(any(JobID.class), anyString()))
+		when(clusterClient.triggerSavepoint(any(JobID.class), nullable(String.class)))
```

Comment:
@zentol What is the preference between `nullable(<any>)` and `any(<class>)`? Inconsistent on this, take a look at `HadoopOutputFormatTest.java` where change `any(FileSystem.class)` to `nullable(FileSystem.class)`